container/blat/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:bullseye-slim
```

Comment:
I think we should use `bookworm-slim`, just like we do in `container/deps/Dockerfile`, we may be able to reuse a cached image layer. Also, `bullseye` is old stable, `bookworm` is current stable.

Reply:
Changed.